pages/examples/checklist-plots.py

```diff
@@ -0,0 +1,58 @@
+import plotly.express as px
+from dash import Dash, dcc, html
```

Comment:
This can go on one line:
from dash import Dash, dcc, Input, Output

* no need to import html since we'll be using dbc for the gallery apps